sjb/hack/determine_install_upgrade_version.py

```diff
@@ -96,7 +102,10 @@ def sort_pkgs(available_pkgs):
 	#          ORIGIN_UPGRADE_RELEASE_MINOR_VERSION=5
 	#
 	if args.dependency_branch.startswith("release") or args.dependency_branch.startswith("enterprise"):
-		pkg_version = args.dependency_branch.split("-")[1]
+		input_pkg_version_parsed = pkg_version.split(".")
+		dependency_branch_version_parsed = args.dependency_branch.split("-")[1].split(".")
+		dependency_branch_version_parsed[0] = input_pkg_version_parsed[0]
```

Comment:
So in case the input is:
```
python sjb/hack/determine_install_upgrade_version.py atomic-openshift-utils-3.6.142-1.git.0.90b5f60.el7.noarch --dependency_branch release-1.5
```
It will set the "search version" to `3.5` since, cause we need to check for that version and not the `1.5`, for that reason Im setting the major version, which is the `dependency_branch_version_parsed[0]`, to input package major version, in this case `3.5`